src/main/java/org/spongepowered/common/mixin/core/item/MixinItemArmorMaterial.java

```diff
@@ -58,4 +67,23 @@ public String getId() {
     public String getName() {
         return this.name;
     }
+
+    @Override
+    public Optional<ItemType> getRepairItemType() {
```

Comment:
IIRC by the time this was implemented, there was no useful method in vanilla mc. Now there is the `shadow$getRepairItem()` that can be used for that. Fixed in latest commit. 

But it does not really matter since it is an enum.